frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -0,0 +1,58 @@
+import { OmegaUp } from '../omegaup';
+import * as time from '../time';
+import { types } from '../api_types';
+import * as api from '../api';
+import * as ui from '../ui';
+import Vue from 'vue';
+import arena_ContestPractice, {
+  ActiveProblem,
+} from '../components/arena/ContestPractice.vue';
+
+OmegaUp.on('ready', () => {
+  time.setSugarLocale();
+  const payload = types.payloadParsers.ContestPracticePayload();
+  const problemInfo = payload.problem;
+  const problem: ActiveProblem | null = payload.problem
+    ? { runs: payload.problem.runs ?? [], alias: payload.problem.alias }
+    : null;
```

Comment:
```suggestion
  const problem: ActiveProblem | null = payload.problem ?? null;
```

bajo qué circunstancia `payload.problem.runs` podría ser nulo? según la definición, nunca.

Reply:
Tienes razón, no debería ser nulo en ninguna circunstancia. De cualquier forma, este código ya desapareció con los últimos cambios que propuse.